python/requirements/ml/dl-cpu-requirements.txt

```diff
@@ -6,25 +6,27 @@ tensorflow-macos==2.15.1; sys_platform == 'darwin' and platform_machine == 'arm6
 tensorflow-probability==0.23.0
 tensorflow-io-gcs-filesystem==0.31.0
 tensorflow-datasets
-array-record==0.5.0; sys_platform != 'darwin' and platform_system != "Windows"
-etils==1.3.0
+array-record==0.5.1; sys_platform != 'darwin' and platform_system != "Windows"
+etils==1.5.2
 
 # If you make changes below this line, please also make the corresponding changes to `dl-gpu-requirements.txt`
 # and to `install-dependencies.sh`!
 
 --extra-index-url https://download.pytorch.org/whl/cpu  # for CPU versions of torch, torchvision
---find-links https://data.pyg.org/whl/torch-1.13.0+cpu.html  # for CPU versions of torch-scatter, torch-sparse, torch-cluster, torch-spline-conv
---find-links https://data.pyg.org/whl/torch-2.0.1+cpu.html  # for CPU versions of torch-scatter, torch-sparse, torch-cluster, torch-spline-conv
+--find-links https://data.pyg.org/whl/torch-2.3.0+cpu.html  # for CPU versions of torch-scatter, torch-sparse, torch-cluster, torch-spline-conv
 
-torch==2.0.1
+torch==2.3.0; sys_platform != 'darwin'
+torch==2.0.1; sys_platform == 'darwin'
```

Comment:
delta comparing to https://github.com/ray-project/ray/commit/13d02a3c198279ceffcd9e40a516f854dd714f73, since latest torch doesn't build for macos x86_64 anymore, we need to pin the version for linux only